% !TeX root = ../../manuscript.tex
\section{Introduction}

Proteins are ubiquitous building blocks playing a critical role in the reproduction, metabolism, and regulation of living organisms and viruses. Understanding and manipulating the way proteins interact with their surrounding is, therefore, of utmost interest from both a biological and a medical point of view. Currently, the most important method to manipulate the function of proteins is through the administering of drugs. For this reason, there exist a growing interest in identifying new binders for a wide variety of proteins in the hopes of treating a number of different sicknesses \cite{24clusters-vina,clusters_vina2,ensemble_screening,autodock_screen,tubercolosis_screen,cancer_inhibitor, vina_theory_practice}. In fact, 78 \% of the biological drugs approved by the United States Food and Drug Administration (FDA) have clear protein molecular targets \cite{how_many_targets}. Therefore, it is not surprising that scientists turned to them once again, when faced with the new and immediate challenges of the coronavirus disease 2019 (COVID-19) pandemic.    

The COVID-19 pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continues to claim thousands of lives every day more than a year after its outbreak \cite{owidcoronavirus}.
However, the knowledge about it and the developed tools to fight against it are vastly more potent than they were a year before \cite{covid_review}.
Antiviral drugs targeting the proteins vital to the reproduction of SARS-CoV-2 have been the most important tools, aside from vaccines which can only be used as preventative measures.
For example, Remdesivir, one of the most widely used antiviral drugs against SARS-CoV-2 around the world \cite{remdesivir_meta}, targets the RNA-dependent RNA polymerase (RdRp) protein of the virus \cite{remdesivir_final}.
Furthermore, given the urgency of developing an effective treatment, most attempts to find new inhibitor substances were in fact drug repurposing studies, targeting the virus's RdRp \cite{rdrp,rdrp_virtual_screening,rdrp_virtual_screening2,rdrp_virtual_screening3,ml_inhibitors} or other important proteins \cite{ensemble_3clpro,main_protease_vs,3covid_proteins,papain-like}.
The RdRp protein is an especially promising drug target as it is responsible for the replication of the viral RNA inside the host cell \cite{rdrp_model}, and it is highly similar to the RdRp of SARS-CoV \cite{hcq_inhibitor}, which already has a number of verified inhibitors \cite{covid_proteins}. In addition, its high-quality three dimensional (3D) structure has been available from as early as April 2020 \cite{rdrp_structure}. In large part due to the urgent nature of the COVID-19 pandemic, most of the above cited research projects relied heavily, or even exclusively, on computational techniques for the discovery of the potential inhibitors, due to the cost and time efficiency of such methods.

High-throughput screening enables the routinely evaluation of thousands of substances in a week \cite{hts}. This tremendous efficacy is often supported by the development and application of innovative computational methods, which became more useful since the advent of structure-based drug design, where potential drugs are created or found based on the 3D structure of the protein target \cite{drug_design_book,structure-based}. Although such target structures were initially only obtainable through costly and cumbersome experimental methods, such as X-ray crystallography \cite{xray} or nuclear magnetic resonance (NMR) spectroscopy \cite{nmr}, they are nowadays much more readily available due to the gradual improvement of existing methods, the appearance of new experimental methods, such as cryo-electron microscopy \cite{cryo-electron}, and the development of recent computational techniques, such as homology modeling \cite{genome3d}. Taking advantage of the quickly growing body of available genomic data, computational tools capable of predicting protein structures from mere amino acid sequence information have also been developed \cite{ensembler, alphafold}. By employing one (or a combination) of the above techniques, high-quality structures are available for a larger number of protein targets than ever before.

The current challenge to computational chemists is therefore how to best utilise the available structural information.
The computational methods developed for structure-based drug design fall into two main categories: {\it de novo} design methods construct new, tailored ligands, while docking methods select ligands complimentary to the target from the existing compound space \cite{docking_to_ensemble}.
Among the docking methods, virtual screening (VS) has emerged as a particularly successful technique \cite{vina_theory_practice, virtual_screening}. This procedure can be thought of as a computational extension to high-throughput screening, where a large number of compounds are docked to the target protein structure {\it in silico}. Traditionally, VS campaigns have been carried out utilising a single, experimentally determined protein structure, often in the crystallised form \cite{docking_to_ensemble,ensemble_discovery}. However, the deficiencies of using only a single crystallised protein structure has been recently recognised \cite{docking_to_ensemble,ensemble_discovery,enrichment,protein_flexibility,protein_flexibility2}. Firstly, the structure of the crystallised protein often differs significantly from the conformations that the protein adopts {\it in vivo}. Secondly, even if the crystal structure is representative of the conformation most often visited in solution, a single structure cannot account for the dynamics of protein motion. 

Different theoretical models that consider the importance of protein motion have been developed, {\it e.g.}, the induced-fit model of ligand docking \cite{first_induced-fit,induced-fit}, where the structure of the protein may change during ligand uptake, or the model of conformational selection \cite{conf_selection1,conf_selection2,cosolvent_md}, which views the target protein as a dynamic object even in the absence of ligands.    The need to take protein flexibility and motion into account became even clearer with the discovery of cryptic or hidden pocket structures \cite{kras,understanding_crypt,cryptic_review}.
The characteristic property of these pockets is that they only appear in the presence of the appropriate ligand, while their existence is not obvious from the equilibrium structure of the protein.
The exact mechanism of their formation is not yet clear, although some combination of induced-fit and conformational selection has been hypothesised \cite{understanding_crypt}.
The discovery and theoretical description of such pockets are hindered by the fact that their opening often requires large scale rearrangements of the protein structure, events that are traditionally hard to predict with computational techniques~\cite{cryptosite}.

With the importance of protein dynamics gaining wider recognition, new, more elaborate methods are appearing which aim to account for this phenomenon. On the one hand some of the modern computational docking programs, such as AutoDock Vina \cite{vina}, can treat a selected number of protein residues as flexible at the cost of increased calculation times.
This method is well suited to study a previously known, specific binding site of the protein. However it cannot account for larger structural changes of the protein and is limited to a handful of flexible residues due to its computational requirements. On the other hand, the family of ensemble docking techniques utilises traditional (rigid protein) docking calculations in combination with an ensemble of protein conformations to account for the flexibility of the target \cite{docking_to_ensemble,ensemble_discovery}.
The careful selection of the structures of the ensemble can enable the description of large scale conformational changes and to the discovery of new cryptic pockets \cite{cryptosite,cosolvent_md,cryptic_review}.
The main challenge for these methods is the generation of the protein structure ensemble, which can be achieved experimentally by using different crystallised structures \cite{docking_to_ensemble,crystal_ensemble1,crystal_ensemble2} or computationally by, {\it e.g.}, conformational space searches \cite{rosettarelax}, neural networks \cite{elastic_NN_ensemble} and molecular dynamics (MD) \cite{ensemble_discovery,md_for_ensemble}. 

MD is an especially promising avenue, after all it has been designed for the very purpose of efficiently sampling the realistic conformational space of proteins. However, one of the largest obstacle of MD calculations is the extremely slow convergence of the calculated trajectories \cite{ensemble_discovery}, which precludes the population of rarely visited conformations. Even with highly specialised code and computers the longest timescales reachable are in the range of milliseconds \cite{anton}. In order to be able to sample rare events, a number of modified MD techniques have been developed. The first group of these is the enhanced sampling methods, where some unphysical bias is introduced into the simulation in order to encourage the sampling of otherwise unlikely conformations. Some of the most popular enhanced-sampling methods in the context of cryptic pocket discovery are umbrella sampling \cite{umbrella}, steered MD \cite{steered_md}, metadynamics \cite{metadynamics}, and replica exchange MD \cite{replica_exchange}, among others. A completely separate approach for the sampling of rarely visited conformations harboring cryptic pockets is that of the cosolvent methods. The main idea behind these frameworks is to replace the traditional water solvent in MD simulations with a mixture of water and some other cosolvent.
The oftentimes hydrophobic or amphipatic cosolvent probes can then interact with the protein and occasionally induce conformational changes or stabilise some conformations where a cryptic pocket is open.
Cosolvent methods have been successfully used to identify cryptic sites in a number of targets \cite{cosolvent_framework,cosolvent_md,cosolvent_based,cryptic_review}.

The primary aim of the presented work is investigate the effect of protein dynamics in the results of a VS campaign. The ensemble of protein structures is obtained via MD simulations. Further sampling is obtained by cosolvent trajectories where water/benzene and water/phenol mixtures are employed. Recognising the severity of the COVID-19 pandemic, the calculations are carried out on the RdRp protein of SARS-CoV-2 and a set of FDA approved small molecule drugs, in the hopes of contributing to the generation of knowledge necessary to develop effective treatments against this virus. 